<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshRenderControlWidget</class>
 <widget class="QWidget" name="MeshRenderControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeshRenderControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Mesh_Boundary">
     <property name="title">
      <string>网格边界</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_Boundary_Faces_Indices">
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_Boundary_Edges">
        <property name="text">
         <string>边</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_Boundary_Edges_Indices">
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_Boundary_Vertices_Indices">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_Boundary_Vertices">
        <property name="text">
         <string>点</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_Boundary_Faces">
        <property name="text">
         <string>面</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_Boundary_Cells">
        <property name="text">
         <string>体</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_Boundary_Cells_Indices">
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Mesh_Inner">
     <property name="title">
      <string>网格内部</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_Inner_Vertices">
        <property name="text">
         <string>点</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_Inner_Vertices_Indices">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_Inner_Edges">
        <property name="text">
         <string>边</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_Inner_Edges_Indices">
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_Inner_Faces_Indices">
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_Inner_Faces">
        <property name="text">
         <string>面</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_Inner_Cells">
        <property name="text">
         <string>体</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_Inner_Cells_Indices">
        <property name="text">
         <string>序号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
